### PR TITLE
Move webroot from project.json to config.

### PR DESCRIPTION
**Pull request by Tratcher**
#411 @davidfowl @muratg

### CODE REVIEW

File: test/Microsoft.AspNet.TestHost.Tests/TestServerTests.cs [hunk 1]
```diff
@@ -357,12 +357,12 @@ public ContextHolder(IHttpContextAccessor accessor)
...
 
         [Fact]
-        public void WebRootCanBeResolvedWhenNotInTheProjectJson()
+        public void WebRootCanBeResolvedWhenNotInTheConfig()
         {
             TestServer server = TestServer.Create(app =>
             {
                 var env = app.ApplicationServices.GetRequiredService<IHostingEnvironment>();
-                Assert.Equal(Directory.GetCurrentDirectory(), env.WebRootPath);
+                Assert.Equal(Directory.GetCurrentDirectory() + "\\wwwroot", env.WebRootPath);
```
Comment (davidfowl) [inline test/Microsoft.AspNet.TestHost.Tests/TestServerTests.cs]: Path.Combine for xplat

**Comment by Tratcher** [comment]
Minor test update.

**Comment by davidfowl** [comment]
Why does travis hate this?

**Comment by davidfowl** [comment]
One behavior change is that we will always create a wwwroot. I think we shouldn't do that. We need to only do it if there is a wwwroot. So the create if not exists logic only runs if it is defined, we can still have a default wwwroot anyways.

**Comment by Tratcher** [comment]
You mean default WebRootPath to applicationBasePath instead of applicationBasePath\wwwroot if webroot isn't specified in the config?

**Comment by davidfowl** [comment]
@Tratcher That's what we did before right?

**Comment by Tratcher** [comment]
Yes. Updated.

**Comment by davidfowl** [comment]
:shipit: tho I dunno why the ci is failing

**Pull request merged**